solidity/dashboard/src/components/copy-stake-steps/Step1.jsx

```diff
@@ -0,0 +1,127 @@
+import React, { useEffect } from "react"
+import TokenAmount from "../TokenAmount"
+import Button from "../Button"
+import { KeepLoadingIndicator } from "../Loadable"
+import { colors } from "../../constants/colors"
+
+const styles = {
+  title: {
+    marginBottom: "1rem",
+  },
+  tileTitle: {
+    textAlign: "justify",
+    marginBottom: "2rem",
+  },
+  addressWrapper: {
+    marginBottom: "0.5rem",
+  },
+  delegationsList: { width: "100%" },
+  delegationItem: {
+    borderRadius: "10px",
+  },
+  selectedDelegationItem: {
+    borderRadius: "10px",
+    border: `1px solid ${colors.secondary}`,
+  },
+}
+
+const CopyStakeStep1 = ({
+  fetchDelegations,
+  isFetching,
+  delegations,
+  decrementStep,
+  incrementStep,
+  onSelectDelegation,
+  selectedDelegation,
+}) => {
+  useEffect(() => {
+    fetchDelegations()
+  }, [fetchDelegations])
+
+  return (
+    <>
+      <h2 style={styles.title}>Stake balances to be upgraded.</h2>
```

Comment:
Is there a reason we mention `stake balances`? Wouldn't `stake delegations` or just `stakes` be more accurate?

```suggestion
      <h2 style={styles.title}>Stake delegations to be upgraded.</h2>
```

```suggestion
      <h2 style={styles.title}>Stakes to be upgraded.</h2>
```

Reply:
Implemented according to Figma views 🤷‍♂️